doc/common-issues.rst

```diff
@@ -0,0 +1,101 @@
+Frequently Encountered Issues
+=============================
+
+.. contents::
+
+Server reports wire version 5, PyMongo requires 6
+-------------------------------------------------
+
+When one attempts to connect to a <=3.4 version server, PyMongo will throw the following error:
+
+.. code-block::
+
+	>>> client.admin.command('ping')
+	...
+	pymongo.errors.ConfigurationError: Server at localhost:27017 reports wire version 5, but this version of PyMongo requires at least 6 (MongoDB 3.6).
+
+This is caused by the driver being too new for the server it is being run against.
+To resolve this issue either upgrade your database to version >= 3.6 or downgrade to PyMongo 3.x which supports MongoDB >= 2.6.
+
+
+'Cursor' object has no attribute '_Cursor__killed'
+--------------------------------------------------
+
+On versions of PyMongo <3.9, when supplying invalid arguments the constructor of Cursor,
+there will be two exceptions raised: AttributeError and TypeError. The AttributeError is not relevant,
+instead look at the TypeError for debugging information.
+
+.. code-block::
+
+	>>> Cursor(wrong=1)
```

Comment:
let's show this using the find() method since users do not call Cursor directly.